### ISSUE TITLE
Token regeneration results

### ISSUE THREAD

**Issue by edward3862** [subrtadel/DIA#5]
Hello, I'm running the provided script for CLIP feature estimation. When I regenerate the images by setting `regenerate_tokens`, I get the following results in `token_regeneration_with_ref.png`.

<img width="869" alt="image" src="https://github.com/subrtadel/DIA/assets/16095941/e54725b9-9c99-4485-a564-edba48006950">

<img width="869" alt="image" src="https://github.com/subrtadel/DIA/assets/16095941/f999000f-bf25-4bf8-8aee-23daeb7579e8">

I'm not sure whether this meant to be correct. In my understanding, the regenerated images are supposed to contain similar content with the input image. Is there anything I missed and how should I fix this issue? 

Thanks!

**Comment by subrtadel** [comment]
You are right, the images should contain similar content. Unfortunately, the CLIP feature estimation is very sensitive to hyperparameter setting. I would try increasing the number of optimization steps (conditioning_optimization.opt_iters) in parameter_estimation.yaml.

**Comment by dsubias** [comment]
Hello,

What is the number of "opt_iters" used to obtain the results shown in the teaser of the article? 

Thanks!
